README.md

```diff
@@ -1,18 +1,20 @@
 # ConfigCat SDK for Elixir
 
 https://configcat.com
+
 ConfigCat SDK for Elixir provides easy integration for your application to ConfigCat.
 
-ConfigCat is a feature flag and configuration management service that lets you separate releases from deployments. You can turn your features ON/OFF using <a href="http://app.configcat.com" target="_blank">ConfigCat Dashboard</a> even after they are deployed. ConfigCat lets you target specific groups of users based on region, email or any other custom user attribute.
+ConfigCat is a feature flag and configuration management service that lets you separate releases from deployments. You can turn your features ON/OFF using [ConfigCat Dashboard](http://app.configcat.com) even after they are deployed. ConfigCat lets you target specific groups of users based on region, email or any other custom user attribute.
```

Comment:
I believe this change was intentional. Like this, the link opens on the same tab by default and I think they wanted to use `target="_blank"` because of its behavior.

Reply:
I wondered about that, but these look really bad in HexDocs, because it doesn't handle literal `a` tags.  We could make a copy of the README for use in the HexDocs, but that's an additional maintenance burden.

This same consideration arises with your comment below about `media` vs `assets`. We can put the image in a `media` folder and specify that ex_doc should copy it to the `assets` folder in the docs (we have no choice about the name there), but then the reference to it in the README then only works in once place or the other.

@laliconfigcat What do you want to do here? Do we want to maintain consistency with the other SDKs at the expense of having to maintain a second almost-identical document? Or is it OK to have a couple of relatively minor differences in order to work better with Elixir's standard tools here?

Reply:
It's fine for us without the <a> tags and also the media -> assets folder